src/main/java/org/amahi/anywhere/util/FingerprintCheck.java

```diff
@@ -0,0 +1,79 @@
+package org.amahi.anywhere.util;
+
+import android.Manifest;
+import android.app.Activity;
+import android.content.Context;
+import android.content.Intent;
+import android.content.pm.PackageManager;
+import android.hardware.fingerprint.FingerprintManager;
+import android.os.Build;
+import android.os.CancellationSignal;
+import android.support.annotation.RequiresApi;
+import android.support.v4.app.ActivityCompat;
+import android.support.v4.content.ContextCompat;
+import android.widget.TextView;
+
+import org.amahi.anywhere.R;
+import org.amahi.anywhere.activity.NavigationActivity;
+
+
+/**
+ * Created by whit3hawks on 11/16/16.
+ */
+@RequiresApi(api = Build.VERSION_CODES.M)
+public class FingerprintCheck extends FingerprintManager.AuthenticationCallback {
```

Comment:
`FingerprintManager.AuthenticationCallback` this class is deprecated. Please update to the currently used implementation.